Sources/Core/AWSClientRuntime/Signing/AWSSigV4Signer.swift

```diff
@@ -9,7 +9,89 @@ import AwsCommonRuntimeKit
 import ClientRuntime
 import Foundation
 
-public class AWSSigV4Signer {
+public class AWSSigV4Signer: ClientRuntime.Signer {
+    public typealias IdObj = Credentials
+
+    public func sign(
+        requestBuilder: SdkHttpRequestBuilder,
+        identity: Credentials,
+        signingProperties: ClientRuntime.Attributes
+    ) async throws -> SdkHttpRequestBuilder {
+        guard let isBidirectionalStreamingEnabled = signingProperties.get(
+            key: AttributeKeys.bidirectionalStreaming
+        ) else {
```

Comment:
if its short enough preferred swift spacing I believe is to have it on one line other than the error handling

```
guard let isBidirectionalStreamingEnabled = signingProperties.get(key: AttributeKeys.bidirectionalStreaming) else {
    throw ClientError.authError("Signing properties must contain a flag for bidirectional streaming.")
}
```

Reply:
Originally had it that way, but swiftlint complains about length - so had to shorten it this way